src/language/tags.md

```diff
@@ -37,3 +37,7 @@ Tags such as `img`, `meta`, and `link` are automatically self-closing (unless yo
 foo/
 foo(bar='baz')/
 ```
+
+## Rendered Whitespace For Inline Elements
+
+Inline (as opposed to block-level) HTML elements --- that is, elements that only contain ["phrasing content"](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content) --- behave like [plain text](plain-text.html#whitespace-control) in Pug: By default, whitespace is removed at the beginning and end of the line.
```

Comment:
Yes, we strongly recommend against pretty mode because of exactly this inconsistency.  It's not clear what is going to be block and what is going to be inline.  A page's style sheet can perfectly legitimately say:

```css
div {
  display: inline;
```

In which case, it may become important to remove the white space between two divs.  If you have pretty mode enabled, this is impossible to do.  It is also very un-safe to swap between pretty mode and non-pretty mode, because it tends to create subtle bugs due to different whitespace behaviour.  We generally recommend using the Elements tab in chrome dev tools (or equivalent in your chosen browser) instead of directly reading the html text that pug outputs.